docs/meshstack.azure.landing-zones.md

```diff
@@ -62,13 +63,20 @@ When parameters are marked as static in the Azure Panel, they can not be replace
   <figcaption>Static Parameter usage in Azure Blueprint Panel</figcaption>
 </figure>
 
-#### Max. Auto Upgrade Blueprint Version
+### Max. Auto Upgrade Blueprint Version
 
 Blueprints are versioned in Azure and can be managed via the Azure Portal. To avoid the accidental assignment of new (and possibly faulty) Blueprints there is this `Max. Auto Upgrade Blueprint Version` field. If you enter a version identifier here which corresponds to a existing Blueprint version in the Azure portal:
```

Comment:
```suggestion
Blueprints are versioned in Azure and can be managed via the Azure Portal. To avoid the accidental assignment of new (and possibly faulty) Blueprints, meshcloud offers a `Max. Auto Upgrade Blueprint Version` field. If you enter a version identifier here which corresponds to a existing Blueprint version in the Azure portal:
```

Reply:
official wording is always meshStack (to not confuse the product with the company).